cumulus/parachains/integration-tests/emulated/assets/asset-hub-westend/src/tests/swap.rs

```diff
@@ -150,20 +151,21 @@ fn swap_locally_on_chain_using_foreign_assets() {
 		(sov_penpal_on_asset_hub_westend.clone().into(), 1000_000_000_000_000_000 * WESTEND_ED),
 	]);
 
-	let sov_penpal_on_asset_hub_westend_as_location: MultiLocation = MultiLocation {
+	let sov_penpal_on_asset_hub_westend_as_location: Location = Location {
 		parents: 0,
-		interior: X1(AccountId32Junction {
+		interior: [AccountId32Junction {
 			network: None,
 			id: sov_penpal_on_asset_hub_westend.clone().into(),
-		}),
+		}]
+		.into(),
 	};
 
 	let call_foreign_assets_create =
 		<AssetHubWestend as Chain>::RuntimeCall::ForeignAssets(pallet_assets::Call::<
 			<AssetHubWestend as Chain>::Runtime,
 			Instance2,
 		>::create {
-			id: *foreign_asset1_at_asset_hub_westend,
+			id: *foreign_asset1_at_asset_hub_westend.clone(),
```

Comment:
```suggestion
			id: foreign_asset1_at_asset_hub_westend.clone(),
```

Reply:
The `*` for de-referencing is needed since we get the contents of the box and then clone those